content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+
+The functionality resides in mutiple tidymodels packages. The easiest way to install them all is to install the tidymodels meta-pacakge:
+
+```{r, eval = FALSE}
+install.packages("tidymodels")
+```
+
+This blog post will highlight why this is useful, explain which additions we've made to the framework, and point to several places to learn more.
+
+You can see a full list of changes in the release notes:
+
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-120)
+- [censored](https://censored.tidymodels.org/news/index.html#censored-030)
+- [yardstick](https://yardstick.tidymodels.org/news/index.html#yardstick-130)
+- [workflows](https://workflows.tidymodels.org/news/index.html#workflows-114)
+- [tune](https://tune.tidymodels.org/news/index.html#tune-120)
+- [finetune](https://finetune.tidymodels.org/news/index.html#finetune-120)
+- [workflowsets](https://workflowsets.tidymodels.org/news/index.html#workflowsets-110)
+
+
+## Increasing usefulness: Two perspectives
+
+We'd like to situate the changes from two different perspectives: How this is useful for people already familiar with survival analysis as well as for people already familiar with tidymodels. 
+
+If you are already familiar with both: Excellent, this is very much for you! Read on for more details on how these two things come together.
+
+### Add tidymodels to your tool kit
+
+If you are already familiar with survival analysis but maybe not tidymodels, these changes now unlock a whole framework for predictive modelling for you. It applies tidyverse principles to modeling, meaning it strives to be consistent, composable, and human-centered. The framework covers the modeling process from the initial test/train split of the data all the way to tuning various models. Along the way it offers a rich selection of preprocessing techniques, resampling schemes, and performance metrics along with safe-guards against accidental overfitting. We make the full case for tidymodels at [tidymodels.org](https://www.tidymodels.org/).
+
+### Add survival analysis to your tool kit
+
+If you are already familiar with tidymodels but maybe not survival analysis, these changes let you leverage the familiar framework for an additional type of modeling problem. Survival analysis offers methods for modeling time-to-event data. While it has its roots in medical research, it has broad applications as that event of interest can be so much more than a medical outcome. Take customer churn as an example: We are interested in how long someone is a customer for and when they churn. For customers who churned, we have the complete time for which they were customers. For existing customers, we only know how long they've been customers for _so far_. Such observations are called censored. So what are our modeling choices here? 
```

Comment:
Would an explicit mention of `mode = "censored regression"` in addition to `"regression"` and `'classification"` be helpful here?